### PR TITLE
Add ci-unix-static GitHub workflow

### PR DESCRIPTION
**Pull request by y-guyon**
Inspired from https://github.com/AOMediaCodec/libavif/blob/main/.github/workflows/ci-unix-static.yml

### CODE REVIEW

File: sys/dav1d-sys/build.rs
Comment (vigneshvg) [inline sys/dav1d-sys/build.rs:1]: i just updated this file to fix the local builds. so this change should not be necessary.

**Comment by vigneshvg** [comment]
[787224d](https://github.com/webmproject/CrabbyAvif/commit/787224db12d6933b58d990b3f1ef878e5eaee35f) should fix the CI failures that are happening.

File: sys/dav1d-sys/build.rs
Comment (y-guyon) [reply sys/dav1d-sys/build.rs:1]: Reverted. By curiosity why leave other `println` but not this one?

Comment (vigneshvg) [reply sys/dav1d-sys/build.rs:1]: https://doc.rust-lang.org/cargo/reference/build-scripts.html#outputs-of-the-build-script

the outputs of the build script are parsed by cargo to determine the compiler and linker flags.

**Review by vigneshvg — APPROVED** [review]

**Pull request merged**